en/docs/develop/creating-artifacts/data-services/creating-data-services.md

```diff
@@ -35,37 +35,56 @@ project as show below.
 
 You can configure the datasource connection details using this section.
 
-1.	Click **Data Sources** to expand the section.
-
+1.	Open the **DDS Service Designer** and click on **Edit** to reopen the **Data Service Form**.
 	![]({{base_path}}/assets/img/integrate/tutorials/data_services/add-datasource-1.png)
 
-2.	Click **Add New** to open the **Create Datasource** page.
-
+2.	Click on the **+ Add Datasource** to open the **Create Datasource** form.
 	![]({{base_path}}/assets/img/integrate/tutorials/data_services/add-datasource-2.png)
 
 3.	Enter the datasource connection details.
-4.	Click **Test Connection** to expand the section.
 
-    ![]({{base_path}}/assets/img/integrate/tutorials/data_services/test_connection.png)
+4. Click **Add** save and add the datasource configurations to the data service.
+
+### Advanced properties 
+
+Click **Advanced Configurations** to expand the section and add the required configurations.
+
+![]({{base_path}}/assets/img/integrate/tutorials/data_services/advances_properties_expanded.png)
+
+The data service should now have the query element added.
 
-5.  Click the **Test Connection** button to verify the connectivity between the MySQL datasource and the data service.
+### Adding a SOAP operation
```

Comment:
Let's move REST resource section before SOAP. We can add soap operation section as a note